com.unity.cinemachine/Runtime/Behaviours/CinemachineGroupFraming.cs

```diff
@@ -133,11 +133,17 @@ class VcamExtraState : VcamExtraStateBase
             public Vector2 RotAdjustment;
             public float FovAdjustment;
 
+            public CinemachineCore.Stage Stage = CinemachineCore.Stage.Finalize; // uninitialized state
+            public CinemachineConfiner2D Confiner;
+            public float PreviousOrthoSize;
```

Comment:
```suggestion
            [SerializedField]
            private float m_PreviousOrthoSize;
```
This is adding API. If we do that we need a minor bump not a patch. Can we make it private or internal instead? Maybe we could start planning for 3.2?

Reply:
It's not new API, it's an internal private class

Reply:
Oh ok yeah mb